docs/guide/advanced/accessibility.md

```diff
@@ -0,0 +1,3 @@
+# Accessibility
+
+<!-- Description how you should use svg icons keeping web accessible -->
```

Comment:
Since this is no added right now I just wanted to drop a few links if there is more research needed:

[Accessible SVG Icons](https://css-tricks.com/accessible-svg-icons/)

Also from the W3C:

> The accessible name is the name of a user interface element. Each platform [accessibility API](https://www.w3.org/TR/wai-aria/#dfn-accessibility-api) provides the accessible name property. The value of the accessible name may be derived from a visible (e.g., the visible text on a button) or invisible (e.g., the text alternative that describes an icon) property of the user interface element. See related [accessible description](https://www.w3.org/TR/wai-aria/#dfn-accessible-description).

[Source](https://www.w3.org/TR/wai-aria/#dfn-accessible-name)

As far as I can see there isn't a lot of info out there about this topic so I would suggest checking how other icons libraries handle this:

[Material Icons](https://www.w3.org/TR/wai-aria/#dfn-accessible-name)
[Font Awesome](https://fontawesome.com/docs/web/dig-deeper/accessibility)

And then there's also more general stuff like enough contrast, big enough size, etc.

Reply:
Awesome tnx!!